osi_sensordata.proto

```diff
@@ -297,4 +298,19 @@ message SensorData
     // object hypothesis and tracking.
     //
     optional FeatureData feature_data = 26;
+
+    // Logical detection data interface.
+    //
+    // Logical detection data are provided by a transformation
+    // (and optional sensor fusion)
+    // performed by a sensor model or a logical Model
+    // that fuses multiple sensors and/or sensor types
```

Comment:
// performed by a sensor model or a logical model
(have both "M"s in model either small or capital letter, prefer small)

Reply:
Yes.